src/app/Dashboard/JvmDetails/JvmDetailsCard.tsx

```diff
@@ -0,0 +1,116 @@
+ * (collectively the "Software"), free of charge and under any and all copyright
+ * rights in the Software, and any and all patent rights owned or freely
+ * licensable by each licensor hereunder covering either (i) the unmodified
+ * Software as contributed to or provided by such licensor, or (ii) the Larger
+ * Works (as defined below), to deal in both
+ *
+ * (a) the Software, and
+ * (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ * one is included with the Software (each a "Larger Work" to which the Software
+ * is contributed by such licensors),
+ *
+ * without restriction, including without limitation the rights to copy, create
+ * derivative works of, display, perform, and distribute the Software and make,
+ * use, sell, offer for sale, import, export, have made, and have sold the
+ * Software and the Larger Work(s), and to sublicense the foregoing rights on
+ * either these or other terms.
+ *
+ * This license is subject to the following condition:
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { FeatureLevel } from '@app/Shared/Services/Settings.service';
+import { Target } from '@app/Shared/Services/Target.service';
+import EntityDetails from '@app/Topology/Shared/Entity/EntityDetails';
+import { NodeType, TargetNode } from '@app/Topology/typings';
+import { useSubscriptions } from '@app/utils/useSubscriptions';
+import { CardActions, CardBody, CardHeader } from '@patternfly/react-core';
+import * as React from 'react';
+import { DashboardCardDescriptor, DashboardCardProps, DashboardCardSizes } from '../Dashboard';
+import { DashboardCard } from '../DashboardCard';
+import '@app/Topology/styles/base.css';
+
+export interface JvmDetailsCardProps extends DashboardCardProps {}
+
+export const JvmDetailsCard: React.FC<JvmDetailsCardProps> = (props) => {
+  const context = React.useContext(ServiceContext);
+  const addSubscription = useSubscriptions();
+
+  const [target, setTarget] = React.useState(undefined as Target | undefined);
+
+  React.useEffect(() => {
+    addSubscription(context.target.target().subscribe(setTarget));
+  }, [addSubscription, context.target, context.api, setTarget]);
+
+  const wrappedTarget = React.useMemo((): TargetNode | undefined => {
```

Comment:
I think we can do this (memo the entire entity instead of just targetNode).

```ts
const wrappedTarget = React.useMemo(() => {
    if (!target) {
      return undefined;
    }
    return {
      getData: (): TargetNode => ({
        name: target.alias,
        target,
        nodeType: NodeType.JVM,
        labels: {},
      })
    };
  }, [target]);

```